pkgs/os-specific/linux/trace-cmd/fix-Makefiles.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/Documentation/Makefile b/Documentation/Makefile
+index 21e42fd..b74c912 100644
+--- a/Documentation/Makefile
++++ b/Documentation/Makefile
+@@ -108,7 +108,9 @@ $(IMGS_INSTALL): %.png.install : %.png force
+ 
+ GUI_INSTALL = $(HTML_INSTALL) $(IMGS_INSTALL)
+ 
+-install: $(MAN1_INSTALL) $(MAN5_INSTALL) $(GUI_INSTALL)
```

Comment:
Why split the targets?

Reply:
For the `trace-cmd` package I only want to install the man pages and not the documentation for kernelshark (`$(GUI_INSTALL)`) so I added the `install_man` target.